kommunicate/src/main/java/io/kommunicate/Kommunicate.java

```diff
@@ -71,7 +73,22 @@ public static void init(Context context, String applicationKey) {
     }
 
     public static void login(Context context, KMUser kmUser, KMLoginHandler handler) {
-        login(context, kmUser, handler, null);
+        if (isLoggedIn(context)) {
```

Comment:
what will happen if someone will pass a different userId?

Reply:
Did the changes for that.